### ISSUE TITLE
Not working on firefox

### ISSUE THREAD

**Issue by iqbalmalik89** [blivesta/animsition#150]
It is not working on firefox.

**Comment by andyduo** [comment]
Yes I see this issue in Firefox and also my android phone browser and chrome mobile. Page loads but the loading animation never clears unless you set timeout: true

**Comment by nOttenga** [comment]
@andyduo you saved my day thx!
